crates/uv/tests/it/pip_install.rs

```diff
@@ -1745,11 +1745,15 @@ fn install_git_public_https_missing_commit() {
     ----- stderr -----
       × Failed to download and build `uv-public-pypackage @ git+https://github.com/astral-test/uv-public-pypackage@79a935a7a1a0ad6d0bdf72dce0e16cb0a24a1b3b`
       ├─▶ Git operation failed
-      ├─▶ failed to clone into: [CACHE_DIR]/git-v0/db/8dab139913c4b566
-      ├─▶ failed to fetch commit `79a935a7a1a0ad6d0bdf72dce0e16cb0a24a1b3b`
-      ╰─▶ process didn't exit successfully: `git fetch [...]` (exit code: 128)
+      ├─▶ failed to find branch, tag, or commit `79a935a7a1a0ad6d0bdf72dce0e16cb0a24a1b3b`
+      ╰─▶ process didn't exit successfully: `/usr/bin/git rev-parse '79a935a7a1a0ad6d0bdf72dce0e16cb0a24a1b3b^0'` (exit code: 128)
+          --- stdout
+          79a935a7a1a0ad6d0bdf72dce0e16cb0a24a1b3b^0
+
           --- stderr
-          fatal: remote error: upload-pack: not our ref 79a935a7a1a0ad6d0bdf72dce0e16cb0a24a1b3b
+          fatal: ambiguous argument '79a935a7a1a0ad6d0bdf72dce0e16cb0a24a1b3b^0': unknown revision or path not in the working tree.
```

Comment:
This error change is a bit of a bummer.

Reply:
Is it actually worse?

Reply:
Well it wasn't great before, but... yeah I think it's worse.

Reply:
I can add some special-casing if you want.

Reply:
I think it's fine as a follow-up, if you want to just open an issue to track it.